man/systemctl.xml

```diff
@@ -876,6 +876,42 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
             a similar filter for messages and might be more
             convenient.
             </para>
+
+
+            <example>
+              <title>Example output from systemctl status </title>
+
+              <programlisting># systemctl status bluetooth
+$ systemctl status bluetooth
+● bluetooth.service - Bluetooth service
+   Loaded: loaded (/lib/systemd/system/bluetooth.service; enabled; vendor preset: enabled)
+   Active: active (running) since Wed 2017-01-04 13:54:04 EST; 1 weeks 0 days ago
+     Docs: man:bluetoothd(8)
+ Main PID: 930 (bluetoothd)
+   Status: "Running"
+    Tasks: 1
+   Memory: 648.0K
+      CPU: 435ms
+   CGroup: /system.slice/bluetooth.service
+           └─930 /usr/lib/bluetooth/bluetoothd
+
+# Recent logs from the bluetooth service would appear here</programlisting>
```

Comment:
I think you could actually provide two or three lines of logs here. We usually do that in such cases to make the example appear as life-like as possible.